presidio-anonymizer/app.py

```diff
@@ -53,13 +55,26 @@ def anonymize():
                 return ErrorResponse("Invalid request json").to_json(), 400
 
             anonymizers_config = AnonymizerRequest.get_anonymizer_configs_from_json(
-                content)
+                content
+            )
             analyzer_results = AnonymizerRequest.handle_analyzer_results_json(content)
-            text = self.engine.anonymize(text=content.get("text"),
-                                         analyzer_results=analyzer_results,
-                                         anonymizers_config=anonymizers_config)
+            text = self.engine.anonymize(
+                text=content.get("text"),
+                analyzer_results=analyzer_results,
+                anonymizers_config=anonymizers_config,
+            )
             return jsonify(result=text)
 
+        @self.app.route("/decrypt", methods=["POST"])
```

Comment:
not sure we need to expose a single text decrypt for the anonymizer, but this can be handled once we implement the full decrupt of an anonymizer result.

Reply:
I was thinking same endpoint for both cases. 
It will receive an optional indices parameter, that will determine how will we handle the text (partially/fully)
WDYT?

Reply:
I dont see a need for a decrypt API for just a single text (you dont need the anonymizer for it). we should discuss offline once the supporting features are in place